components/layout/Search.vue

```diff
@@ -82,68 +173,76 @@
 </template>
 
 <script setup>
```

Comment:
![image](https://github.com/kestra-io/docs/assets/37618489/feeb72d3-19de-4f56-86b9-f51f920415b2)
please fix your IDE to our custom indent to prevent noise on your PRs